src/components/dictionaryConcepts/containers/EditConcept.jsx

```diff
@@ -348,6 +353,20 @@ export class EditConcept extends Component {
     this.hideGeneralModal();
   }
 
+  selectConfirm = () => this.props.history.push(localStorage.getItem('dictionaryPathName'));
+
```

Comment:
A null check should be performed on the localStorage.getItem(...)

Reply:
@paulvazbad, what is the essence of performing a null check on the localStorage.getItem

Reply:
I have refactored and fixed the issue.